SU-HSDO/suhumsci
# SHS-5581: Remove Chosen plugin dependency

# READY FOR REVIEW

## Summary
* Uninstalls `chosen_lib` from Drupal
* Removes chosen CSS class references
* Removes chosen JS references
* Does not remove chosen from `composer.json` as that will take place in a subsequent PR

## Need Review By (Date)
2024-05-16

## Urgency
low

## Steps to Test
1. Login to Drupal and visit `/admin/modules`
2. Verify that Chosen and Chosen Library are both uninstalled
3. Visit a page that has a multi-select box, such as Political Science (`/undergraduate-program/current-courses`), Creative Writing (`/books`), or HS Colorful (`/default-views/courses`)
4. Verify that the multi-select box still works with no visual changes

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
